pkg/deploy/devconfig.go

```diff
@@ -101,6 +101,7 @@ func DevConfig(_env env.Core) (*Config, error) {
 					KeyvaultDNSSuffix:           &_env.Environment().KeyVaultDNSSuffix,
 					KeyvaultPrefix:              &keyvaultPrefix,
 					StorageAccountDomain:        to.StringPtr(os.Getenv("USER") + "aro" + _env.Location() + ".blob." + _env.Environment().StorageEndpointSuffix),
+					DisableOauth:                to.StringPtr("true"),
```

Comment:
Do we need to disable oauth for dev use? I think we have the AAD groups set up for it already, it's just CI that's a problem.

Reply:
ok will try that